lor_deckcodes/decode.py

```diff
@@ -27,10 +35,23 @@ def decode_deck(deckcode: str):
 
     # 3 card copies
     all_cards.extend(_decode_card_block(3, data))
+
     # 2 card copies
     all_cards.extend(_decode_card_block(2, data))
+
     # 1 card copies
     all_cards.extend(_decode_card_block(1, data))
+
+    # more cards
+    n = data.tell()
+    c = data.read(1)
+    data.seek(n)
+    while c:
+        all_cards.extend(_decode_card_block_(data))
```

Comment:
This introduces some code repetition with the called function having the same responsibilities as `_decode_card_block`, can you change the algorithm to use it?